ovs/extensions/services/upstart.py

```diff
@@ -91,18 +92,27 @@ def add_service(name, client, params=None, target_name=None, additional_dependen
             client.file_write('/etc/init/{0}.conf'.format(target_name), template_file)
 
     @staticmethod
-    def get_service_status(name, client):
+    def get_service_status(name, client, return_output=False):
```

Comment:
Well, to be honest; I don't care about the input type enough to justify putting asserts inside the function as in my opinion it makes the code uglier. I'd rather make this type checking decorator we discussed earlier that doesn't mess up the method's body. I'm going to create a ticket for that.